target/linux/mpc85xx/image/p1020.mk

```diff
@@ -15,6 +15,27 @@ define Device/aerohive_hiveap-330
 endef
 TARGET_DEVICES += aerohive_hiveap-330
 
+define Device/aerohive_hiveap-370
```

Comment:
One should consider creating a shared Device/aerohive_hiveap definition here.

Reply:
Is there a good example I can follow ?

Reply:
Something like that, only you can keep it all together and don't have to spread it across files:
https://github.com/openwrt/openwrt/commit/3b89849b57a88d0919b6442509131f045af580a8

Reply:
OK, @adschm; your request is now done:

```diff
@@ -1,13 +1,19 @@                                                                                                                                                                               
-define Device/aerohive_hiveap-330                                                                                                                                                              
+define Device/aerohive_hiveap                                                                                                                                                                  
   DEVICE_VENDOR := Aerohive                                                                                                                                                                    
-  DEVICE_MODEL := HiveAP-330                                                                                                                                                                   
   DEVICE_PACKAGES := kmod-tpm-i2c-atmel                                                                                                                                                        
   BLOCKSIZE := 128k                                                                                                                                                                            
   KERNEL := kernel-bin | gzip | uImage gzip                                                                                                                                                    
-  KERNEL_SIZE := 8m                                                                                                                                                                            
   KERNEL_INITRAMFS := copy-file $(KDIR)/vmlinux-initramfs | uImage none                                                                                                                        
-  IMAGES := fdt.bin sysupgrade.bin                                                                                                                                                             
   IMAGE/fdt.bin := append-dtb                                                                                                                                                                  
+  IMAGES := fdt.bin sysupgrade.bin kernel.bin rootfs.bin                                                                                                                                       
+  IMAGE/kernel.bin := append-kernel                                                                                                                                                            
+  IMAGE/rootfs.bin := append-rootfs | pad-rootfs                                                                                                                                               
+endef                                                                                                                                                                                          
+                                                                                                                                                                                               
+define Device/aerohive_hiveap-330                                                                                                                                                              
+  $(Device/aerohive_hiveap)                                                                                                                                                                    
+  DEVICE_MODEL := HiveAP-330                                                                                                                                                                   
+  KERNEL_SIZE := 8m            
```